static/src/javascripts/projects/common/modules/experiments/tests/remote-render-epic.js

```diff
@@ -43,7 +43,7 @@ const decodeJson = response => response.json();
 const products = ['CONTRIBUTION', 'MEMBERSHIP_SUPPORTER'];
 
 const remoteRenderTest = {
-    id: 'RemoteRenderEpic',
+    id: 'RemoteRenderEpicRoundTwo',
```

Comment:
do we need to change the switch name as well?

Reply:
Yeah sounds sensible. Is that required for the data side, or more about clarity?

Reply:
possibly the test will not run if it cannot be switched on